src/awkward/_v2/contents/listarray.py

```diff
@@ -184,8 +184,11 @@ def _getitem_next(self, head, tail, advanced):
 
             nexthead, nexttail = self._headtail(tail)
 
-            start, stop, step = head.indices(self._stops[0])
+            start, stop, step = head.start, head.stop, head.step
+
             step = 1 if step is None else step
+            start = 0 if start is None else start
+            stop = ak._util.kSliceNone if stop is None else stop
```

Comment:
@ioanaif If `step` is negative, `start is None` is not equivalent to `start == 0`; negative `step` means you're counting down from the length of the list, which isn't a single value that can be replaced here (all of the lists may have different lengths). In situations like this, it should be

```python
            step = 1 if step is None else step
            start = ak._util.kSliceNone if start is None else start
            stop = ak._util.kSliceNone if stop is None else stop
```

I'll do this in the next commit.